modules/installation-gcp-user-infra-shared-vpc-config-yaml.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-user-infra-vpc.adoc
+
+[id="installation-gcp-user-infra-shared-vpc-config-yaml_{context}"]
+= Sample customized `install-config.yaml` file for GCP
+
+You can customize the `install-config.yaml` file to specify more details about your {product-title} cluster's platform or modify the values of the required parameters.
+
+[IMPORTANT]
+====
+This sample YAML file is provided for reference only. You must obtain your `install-config.yaml` file by using the installation program and modify it.
+====
+
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+controlPlane: <2>
+  hyperthreading: Enabled <3> <4>
+  name: master
+  platform:
+    gcp:
+      type: n2-standard-4
+      zones:
+      - us-central1-a
+      - us-central1-c
+  replicas: 3
+compute: <2>
+- hyperthreading: Enabled <3>
+  name: worker
+  platform:
+    gcp:
+      type: n2-standard-4
+      zones:
+      - us-central1-a
+      - us-central1-c
+  replicas: 0
+metadata:
+  name: test-cluster
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16
+  networkType: OpenShiftSDN
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  gcp:
+    ProjectID: openshift-production
+    region: us-central1
```

Comment:
As we define REGION in the Creating the VPC section, please add a note below to instruct that the region should be consistent with REGION. I mean the region in which the cluster will install should be the same as the region that VPC network resides in.